relay-dynamic-config/src/metrics.rs

```diff
@@ -43,7 +43,7 @@ pub struct TagBlock {
     /// Name of metric of which we want to remove certain tags.
     pub name: GlobPatterns,
```

Comment:
Shouldn't this also be `"names"` then?

Reply:
atm, we think of it as it should just be a single name pattern there, it's just the type we use allows for multiple. So, depends on if we should name after what we intend to do, or name after what we are able to do